app.py

```diff
@@ -20,5 +21,28 @@ def Login():
             flash('Login Unsuccessful. Invalid Email/Password')
     return render_template('login.html',title='Login | SAC Portal, IIT Mandi',form=form)
 
+#Example  https://host/leaveclub?userID='B19188'&clubID='C10001'
+@app.route('/leaveclub', methods = ['GET'])
+def leave_club():
+    data = {
+        "tablename" : "ClubMembers",
+        "userID" : request.args.get('userID'),
+        "clubID" : request.args.get('clubID')
+        }
+    stmt = "DELETE FROM ClubMembers WHERE userID='"+data["userID"]+"' and clubID='"+data["clubID"]+"';"
+    #print(stmt)
+    success = 0;msg=''
+    try:
+        mycursor.execute(stmt)
+        mydb.commit()
+        success=1
+        msg = "Deleted"
+    except sqlerror as err:
+        success=0
+        msg = err
```

Comment:
msg = str(err)
the err object here is not json serializable
![Screenshot from 2020-03-31 00-40-21](https://user-images.githubusercontent.com/45260243/77951810-42633b00-72e8-11ea-8918-80d359f91311.png)

Reply:
Yes, I will solve this.